packages/web/pages/pools/index.tsx

```diff
@@ -328,19 +336,45 @@ const Pools: NextPage = observer(function () {
                   }
 
                   return (
-                    <PoolCard
+                    <div
```

Comment:
Again, could we try to avoid this wrap? Perhaps add an onClick handler to `PoolCard`